src/tests/fake/samples.tsx

```diff
@@ -240,7 +240,7 @@ export function mockApiCreateSample(
             subtractions,
             group,
         })
-        .reply(201, sample);
+        .reply(200, sample);
```

Comment:
Is this change based off what the API is currently returning? In theory the API should be returning 201, not sure if in practice it is though